### ISSUE TITLE
Cross-Site Scripting vulnerability in version 2.2.2

### ISSUE THREAD

**Issue by borit22** [webpack-contrib/terser-webpack-plugin#196]
- Operating System: Windows 7 SP1 x64
- Node Version: 12.13.1
- NPM Version: 6.13.2
- webpack Version: 4.41.2
- terser-webpack-plugin Version: 2.2.2

### Expected Behavior

no vulnerability

### Actual Behavior

Moderate        Cross-Site Scripting                                          
                                                                                
Package         serialize-javascript                                          
                                                                                
Dependency of   webpack [dev]                                                 
                                                                                
Path            webpack > terser-webpack-plugin > serialize-javascript  

### Code

```js
"terser-webpack-plugin": "^2.2.2"
```

### How Do We Reproduce?
in webpack.config.js
```js
"terser-webpack-plugin": "2.2.2"
```
and `npm install`

**Comment by borit22** [comment]
[serialize-javascript](https://github.com/yahoo/serialize-javascript) fixed it in version 2.1.1(2.1.2 lastest for now)

**Comment by evilebottnawi** [comment]
Fixed
